src/models/notificationThreads.ts

```diff
@@ -0,0 +1,43 @@
+import { DataTypes, Model } from 'sequelize';
+import { sequelize } from '@/sequelize-instance';
+import type { CreationOptional, InferAttributes, InferCreationAttributes } from 'sequelize';
+
+export class NotificationThread extends Model<InferAttributes<NotificationThread>, InferCreationAttributes<NotificationThread>> {
+	declare id: CreationOptional<number>;
+	declare user_id: string;
+	declare thread_id: string;
+	declare updated: CreationOptional<Date>;
+	declare created: CreationOptional<Date>;
+}
+
+NotificationThread.init({
+	id: {
+		type: DataTypes.INTEGER,
+		autoIncrement: true,
+		primaryKey: true
+	},
+	user_id: {
+		type: DataTypes.STRING
+	},
+	thread_id: {
+		type: DataTypes.STRING
+	},
+	updated: {
+		type: DataTypes.DATE,
+		defaultValue: DataTypes.NOW
+	},
+	created: {
+		type: DataTypes.DATE,
+		defaultValue: DataTypes.NOW
+	}
```

Comment:
They're not used anywhere now, but having them is potentially useful in the future and if we had to add it later we wouldn't have data about the existing threads. Not including it would just save a couple of bytes per entry. 

In fact all of our other models have this as sequelize adds them automatically, however they're not exposed via the TypeScript types for some reason, so I've included them here explicitly and set `timestamps: false` to prevent it auto generating them.

Reply:
You can still get the created date from the Discord thread by checking the thread's `createdAt`. For the updated date, I'm just not sure what that would be useful for. However, I see no harm in keeping them, so if you believe they will be useful it's fine to keep them.

Reply:
Fair, I guess updated is the last message sent. I'm happy to remove these

Reply:
I keep going backwards and forwards on this, I feel like for the tiny overhead it's worth keeping around, @jonbarrow make a decision on this and put me out of my misery 😢

Reply:
Threads can be deleted by anyone with perms in the Discord server, the data in the DB can only be deleted by me. I say keep them